### PR TITLE
Add domain mapping to Apps FAQ

### LINKED ISSUE #254: Add set apps on website root to FAQ
It's only briefly mentioned in the walkthrough.md doc under server settings.

### LINKED ISSUE #265: Document multiple additional domains for apps
https://github.com/btcpayserver/btcpayserver-docker/pull/164
In Server Settings FAQ.

### PR DESCRIPTION
**Pull request by pavlenex**
Closes #265 
Closes #254 
@NicolasDorier is this what you had in mind? I may be missing explanation for .onion? If so what exactly should I add there? I'm still unsure if we need a blog post for this feature as you earlier mentioned.

### CODE REVIEW

File: FAQ/FAQ-Apps.md [hunk 1]
```diff
@@ -303,3 +304,24 @@ However, you can use a work-around:
 
 ![Custom Amount Pay Button](/img/BTCPayPayButtonDynamic2.png)
 ![Custom Amount Pay Button](/img/BTCPayPayButtonDynamic.png)
+
+## How to map a domain name to an app?
+BTCPay Apps can have a domain name that's different from the servers domain. Let's assume you have BTCPay server at mybtcpayserver.com and want to display your PoS app on mybtcpaypos.com instead mybtcpayserver.com/apps/pos/abc123
+First, [configure DNS settings](/ChangeDomain.md#setting-up-your-dns-record) of mypointofsale.com and make sure it's pointing to the external ip of your BTCPay Server.
+
+Next, add additional domain or subdomain name(s) by adding a new enviroment variable through ssh:
+```
```
Comment (NicolasDorier) [inline FAQ/FAQ-Apps.md]:
```bash

Comment (NicolasDorier) [inline FAQ/FAQ-Apps.md]: ````bash`

File: FAQ/FAQ-Apps.md [hunk 2]
```diff
@@ -303,3 +304,24 @@ However, you can use a work-around:
...
+sudo su -
+export BTCPAY_ADDITIONAL_HOSTS="mybtcpaypos.com"
+. ./btcpay-setup.sh -i
+```
+If you want to add multiple domains, you just need to update the env variables again
+```
+sudo su -
+export BTCPAY_ADDITIONAL_HOSTS="mybtcpaypos.com.,subdomain.domain2.com,domain3.com"
```
Comment (NicolasDorier) [inline FAQ/FAQ-Apps.md]: there is an unwanted `.`

**Comment by pavlenex** [comment]
Updated.
Also closes #254

**Comment by NicolasDorier** [comment]
@pavlenex replace `. ./btcpay-setup.sh -i` by `. btcpay-setup.sh -i`.

The former works only if the user is in the right folder, the second works anywhere if the user installed btcpay once.

**Comment by pavlenex** [comment]
Done, I was completely unaware of `. btcpay-setup.sh -i` command. Should we replace the `. ./btcpay-setup.sh -i` with `. btcpay-setup.sh -i` everywhere in docs?

**Comment by NicolasDorier** [comment]
not anywhere. The first install should be `./btcpay-setup.sh` in the right folder. Then `btcpay-setup.sh` can be used anywhere.

**Pull request merged**